Makefile

```diff
@@ -21,7 +21,8 @@ install: fmt REBUILD
 clean:
 	rm -rf hostdir release whitepaper.aux whitepaper.log whitepaper.pdf         \
 		*.wallet* *_test testdir* */*_test hostdir* siad/walletDir*             \
-		siad/hostDir* info.log modules/*/*.dat modules/*/*.backup
+		siad/hostDir* info.log modules/*/*.dat modules/*/*.backup               \
+		/tmp/SiaTesting
```

Comment:
We can once everything has been moved over. The daemon is still creating a bunch of files though. We'll be able to clean out the .gitignore as well.